IBX-9060: Describe notifications in user doc (#368)
ezsystems/user-documentation: docs/getting_started/notifications.md

1. In the top bar, click a notification icon. Then, at the bottom of the short list, click **View all notifications...**.

    Here, you can see a list of notifications, together with information about their status and arrival date and time.

    ![Notifications screen](img/notifications_screen.png "Notifications screen")

    !!! note "List pagination"
    
        The list can take more than one screen. If it is so, you can click the arrows and numbers at the bottom of the screen to navigate the list.

2. Narrow down the list of displayed notifications by selecting one or more of the following filters:

    - **Notification type**: for example, content item review request
    - **Status**: unread or read
    - **Date and time**: start and end of date range within which the notification was received

    ![Notification filters](img/notifications_filters.png "Notification filters")

3. Click **Apply** to confirm your choices.

4. To clear all the filters, click **Clear filters**.

### Act on notifications

In the notification screen, you can act on the notification to fulfill the request of the originator.

> To act on a notification, find it in the notifications list and, in its line, click a link in the title.

Reviewer: There is no link to the title **in the full list** of notifications. Only clicking on the icon will redirect you to the selected content.

> To act on a notification, find it in the notifications list and, in its line, click a link in the title.
> You can also click an icon next to the notifications date and time.

Author: ```suggestion
To act on a notification, find it in the notifications list and click an icon next to the notification's date and time.
```

> Depending on the context, the icon can be called differently, for example, **Go to content**, **Go to trash** or **Accept request**.

Reviewer: Currently there is no "Go to trash"

> Depending on the context, the icon can be called differently, for example, **Go to content**, **Go to trash** or **Accept request**.

Author: ```suggestion
Depending on the context, the icon can be called differently, for example, **Go to content** or **Accept request**.
```
